sql-statements/sql-statement-set-variable.md

```diff
@@ -85,7 +85,11 @@ mysql> SHOW SESSION VARIABLES LIKE 'sql_mode';
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+The following behavior differences apply:
+
+* Changes made with `SET GLOBAL` will be propagated to all TiDB instances in the cluster. This differs from MySQL, where changes do not propagate to replicas.
+* TiDB presents several variables as both readable and settable. This is required for compatibility, since it is common for both applications and connectors to read MySQL variables. For example: JDBC connectors both read and set query cache settings, despite not relying on the behavior.
```

Comment:
```suggestion
* TiDB presents several variables as both readable and configurable. This is required for MySQL compatibility, because it is common for both applications and connectors to read MySQL variables. For example: JDBC connectors both read and configure query cache settings, despite not relying on the behavior.
```

Reply:
I believe it should be 'set' not configure, because this is for the statement 'set'. I will merge in the other suggestions.